student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'unknown', parent_permission: true)
+    super(name, age, parent_permission)
```

Comment:
- [ ] Kindly ensure you are consistent with your parameters. In the person class, which Student inherits from, `parent_permission` is assigned using the equal sign (=), but here you are using the colon notation (:). Kindly choose one and be consistent.

- [ ] In the `#super` method `age` should come before `name`.